clustoapi/server.py

```diff
@@ -306,15 +306,25 @@ def get_from_pools():
     drivers = bottle.request.params.getall('driver')
     children = bottle.request.params.get('children', default=True, type=bool)
     mode = bottle.request.headers.get('Clusto-Mode', default='compact')
+    current = int(bottle.request.headers.get('Clusto-Page', default='0'))
+    per = int(bottle.request.headers.get('Clusto-Per-Page', default='50'))
 
     try:
         ents = clusto.get_from_pools(
             pools, clusto_types=types, clusto_drivers=drivers, search_children=children
         )
         results = []
+        headers = {}
+        if current:
+            ents, total = util.page(list(ents), current=current, per=per)
```

Comment:
Additionally, ignoring the second unpack (like if I took out ``total``) turns the returned object into an actual tuple, is there any way to combat that in the method and return just what is requested to be unpacked? Iono.